src/internal/observable/ConnectableObservable.ts

```diff
@@ -12,8 +12,8 @@ import { refCount as higherOrderRefCount } from '../operators/refCount';
 export class ConnectableObservable<T> extends Observable<T> {
 
   protected _subject: Subject<T>;
-  protected _refCount: number = 0;
-  protected _connection: Subscription;
+  public _refCount: number = 0;
+  public _connection: Subscription;
```

Comment:
We don't want to make these public APIs really.

Reply:
already changed